### PR TITLE
Release 5.1

### PR DESCRIPTION
**Pull request by svx**
This PR adds Plone 5.1

### CODE REVIEW

File: 5.1/5.1.0/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
+FROM python:2.7-alpine
+MAINTAINER "Plone Community" http://community.plone.org
```
Comment (avoinea) [inline 5.1/5.1.0/alpine/Dockerfile]: I think this is deprecated, we should move it to `LABEL`

**Comment by avoinea** [comment]
@svx We should also consider using Docker `HEATHCHECK` instead of our custom made heath-check within entrypoint. But maybe this should be another ticket.

**Comment by svx** [comment]
@avoinea Yeah I did change Maintainer to Label in the Debian images, but I never did that in the Alpine ones :)

Will do that later today :)

**Comment by svx** [comment]
@avoinea 

Yeah +1 we should move to the ``build in`` HEALTHCHECK.

And yes IMHO this should be a own and new ticket !

**Comment by svx** [comment]
@avoinea Updated with https://github.com/plone/plone.docker/pull/81/commits/5ea70cb5cb9ff187b3cc982312f5d7edc11fd8dd

**Comment by avoinea** [comment]
@svx  Ups, https://hub.docker.com/r/plone/plone/builds/bqqw8slamfbxfu6z9pxfzs5/

**Comment by svx** [comment]
@avoinea As far as I know the Alpine build in the Plone repo are broken since a long time

**Comment by avoinea** [comment]
The build was passing before, but indeed, it was failing after #16

**Comment by svx** [comment]
I fixed my own Alpine builds a month or two ago.

I can check what I am doing different sometime this week.

I also would be interested in knowing if people would use Alpine based images at all.
It seems to be most of the people use the Debian based ones.

So should we fix and provide the Alpine ones at all ? :)

**Comment by svx** [comment]
@avoinea Ok actually the build is now stopping with another error before even do anything:
```
./Plone-5.1.0-UnifiedInstaller/install.sh: .: line 734: can't open 'helper_scripts/user_group_utilities.sh'
```

**Comment by svx** [comment]
Digging deeper it seems there is something weird with the UI installer, here is the output if I build the Debian based one:
```
Detailed installation log being written to //install.log
Installing Plone 5.1.0 at /plone

helper_scripts/main_install_script.sh: line 723: helper_scripts/user_group_utilities.sh: No such file or directory
helper_scripts/main_install_script.sh: line 726: check_ug_ability: command not found
helper_scripts/main_install_script.sh: line 728: create_group: command not found
helper_scripts/main_install_script.sh: line 729: create_user: command not found
helper_scripts/main_install_script.sh: line 730: check_user: command not found
helper_scripts/main_install_script.sh: line 731: create_user: command not found
helper_scripts/main_install_script.sh: line 732: check_user: command not found
```
The difference here is that the script keeps running, compared to Alpine (above) where is stops.

@smcmahon did you noticed that before or do you know what is going on here ? :)

**Comment by avoinea** [comment]
@svx I think I did some hacks before, too 
see https://github.com/plone/plone.docker/blob/master/4.3/4.3.15/alpine/Dockerfile#L35

**Comment by svx** [comment]
@avoinea Thanks !

I tried that too :)

Still no luck, maybe my syntax was wrong :)

Did you saw #84 ?

Adding extra downloads just to get the UI working feels wrong to me, for sure if we do not even need that test/check :)

**Pull request merged**